org.BleuBleu.FamiStudio.appdata.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>org.BleuBleu.FamiStudio.desktop</id>
```

Comment:
```suggestion
  <id>org.BleuBleu.FamiStudio</id>
```

Reply:
You need a `<launchable type="desktop-id">org.BleuBleu.FamiStudio.desktop</launchable>` as well